packages/vue/src/T.ts

```diff
@@ -35,7 +39,7 @@ export const T = defineComponent({
       key: this.$props.keyName,
       params: assignedParams,
       defaultValue: this.$props.defaultValue,
-      noWrap: this.$props.noWrap,
+      noWrap: this.isInitialRender || this.$props.noWrap,
```

Comment:
Do we need to do this here? It seems like this should already be solved in VueTolgee.ts

Reply:
You are right. removed.